core/templates/dev/head/components/state-directives/response-header/response-header.directive.ts

```diff
@@ -56,7 +56,7 @@ angular.module('oppia').directive('responseHeader', [
             StateInteractionIdService, INTERACTION_SPECS) {
           var ctrl = this;
           ctrl.EditabilityService = EditabilityService;
-          ctrl.isInQuestionMode = StateEditorService.isInQuestionMode;
+          ctrl.StateEditorService = StateEditorService;
```

Comment:
Ah -- I think you'll need to do `ctrl.isInQuestionMode = function() {...}` to get the original code working.

Reply:
Ok, will do this

Reply:
Done